scripts/lib/cluster.sh

```diff
@@ -95,6 +96,7 @@ function up-kops-cluster {
     --node-size c5.xlarge \
     --ssh-public-key=~/.ssh/devopsinuse.pub \
     --kubernetes-version ${K8S_VERSION} \
+    --image "ssm:/aws/service/canonical/ubuntu/server/20.04/stable/current/amd64/hvm/ebs-gp2/ami-id" \
```

Comment:
nit: can this be a global variable?

Reply:
We are not reusing it anywhere else, but I can update it in a variable